test/ufuzz.js

```diff
@@ -490,20 +497,22 @@ function createStatement(recurmax, canThrow, canBreak, canContinue, cannotReturn
 function createSwitchParts(recurmax, n, canThrow, canBreak, canContinue, cannotReturn, stmtDepth) {
     var hadDefault = false;
     var s = '';
+    canBreak = enableLoopControl(canBreak);
+    if (!canBreak) canBreak = CAN_BREAK;
     while (n-- > 0) {
         //hadDefault = n > 0; // disables weird `default` clause positioning (use when handling destabilizes)
         if (hadDefault || rng(5) > 0) {
             s += '' +
                 'case ' + createExpression(recurmax, NO_COMMA, stmtDepth, canThrow) + ':\n' +
-                createStatements(rng(3) + 1, recurmax, canThrow, canBreak || CAN_BREAK, canContinue, cannotReturn, stmtDepth) +
+                createStatements(rng(3) + 1, recurmax, canThrow, canBreak, canContinue, cannotReturn, stmtDepth) +
                 '\n' +
-                (rng(10) > 0 ? ' break;' : '/* fall-through */') +
+                (rng(10) > 0 ? ' break' + getLabel(canBreak) + ';' : '/* fall-through */') +
```

Comment:
```
 + function getLabel(label) {
 +    if (!Array.isArray(label)) return "";
 +    label = label[rng(label.length)];
 +    return label && " L" + label;
 +}
```
Even if there is a null label in the array - I don't understand the logic in `createLabel()` at all - the more deeply nested it is within labelled loops/blocks the odds of returning a label greatly increase. I'd prefer to keep the chance of generating a label in the presence of labelled loops/blocks to a constant 10% for each case break since they are so frequent.